Lore/world

```diff
@@ -0,0 +1,64 @@
+# Etheria: Realm of ByteBeasts
+
+## World Overview
+
+Etheria is a magical realm where mystical energies flow freely, creating a vibrant medieval fantasy setting. It serves as the backdrop for the ByteBeasts adventures, offering a rich tapestry of history, diverse races, and mysterious creatures.
+
+## Creation Myth
+
+In the beginning, there was the Primordial Ether, a cosmic tapestry of pure magical essence that pulsed with infinite potential. From this mystical source emerged two fundamental forces:
+
+- **Lumina**: Bearers of light and order
+- **Umbra**: Harbingers of shadow and chaos
+
+Their eternal dance wove the fabric of Etheria, a world where magic exists in myriad forms.
+
+## Elemental Essences
+
+Five elemental essences manifested from the interplay of Lumina and Umbra, each corresponding to a type of ByteBeast:
+
+* Crystal ❄️: The Frozen Essence
+* Draconic 🐉: The Wyrm Essence
+* Shadow 🐦‍⬛: The Umbral Essence
+* Light 🌟: The Radiant Essence
+* Titanium ⚙️: The Metallic Essence
+
+These essences breathed life into Etheria, each imbuing a portion of the realm with its unique properties and giving rise to the various types of ByteBeasts.
+
+## ByteBeasts
+
+The first magical entities to emerge were the ByteBeasts, creatures of living magical energy that took on diverse forms reflecting their creator essence. ByteBeasts can bond with Etheria's inhabitants, forming symbiotic relationships that enhance both beings. Each ByteBeast belongs to one of the five elemental types, determining its strengths and abilities.
+
+## Sentient Races
+
+Five primary sentient races arose from the fusion of mortal flesh and elemental magic:
+
+1. **Crystalborn**: Bodies infused with living crystalline magic
+2. **Dracokin**: Descendants of those who formed pacts with ancient dragons
+3. **Shadowwalkers**: Masters of the umbral arts
+4. **Luminary**: Beings woven from pure light
+5. **Ironheart**: Forged from the essence of unyielding metal
+
+## The Ethereal Nexus
```

Comment:
Ethereal Nexus will be the battle zone between players right? In the Town Scene in Unity right? 
It would be amazing if only those players who have already passed the league and their ByteBeasts have reached a certain level can fight here. This encourages exploration and provides exclusivity to the area and the players who have access.